addons/web/static/src/legacy/xml/pivot.xml

```diff
@@ -133,10 +133,10 @@
         </div>
     </t>
 
-    <t t-name="web.legacy.PivotCustomGroupByItem" t-inherit="web.Legacy.CustomGroupByItem" t-inherit-mode="primary" owl="1">
-        <xpath expr="//div[hasclass('o_generator_menu')]" position="attributes">
+    <t t-name="web.legacy.PivotCustomGroupByItem" t-inherit="web.CustomGroupByItem" t-inherit-mode="primary" owl="1">
+        <!-- <xpath expr="//div" position="attributes">
```

Comment:
check what to do

Reply:
If we merge pivot, we can simply remove web.legacy.PivotCustomGroupByItem